deploy-lambda/src/databricks_cdk/job.py

```diff
@@ -25,7 +25,7 @@ class AutoScale(BaseModel):
 class AwsAttributes(BaseModel):
     first_on_demand: int = 1
     availability: str = "SPOT_WITH_FALLBACK"
-    zone_id: str
+    zone_id: Optional[str] = None
```

Comment:
I added this as optional because when deploying a job with an instance pool this needs to be None.

Reply:
👍